admin/services/data-access/check-form-v2.data.service.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+const R = require('ramda')
+const { TYPES } = require('tedious')
+
+const sqlService = require('./sql.service')
+
+const table = '[checkForm]'
+
+const checkFormV2DataService = {
+  /**
+   * Find existing check forms
+   * @returns {Promise<*>}
+   */
+  sqlFindAllCheckForms: async () => {
+    const sql = `
+    SELECT *
+    FROM ${sqlService.adminSchema}.${table}
+    `
+    return sqlService.query(sql)
+  },
+  sqlInsertCheckForms: async (checkFormData, isFamiliarisationCheckFormUpdate) => {
+    const params = []
+    const queries = []
+    const inserts = []
+
+    if (isFamiliarisationCheckFormUpdate) {
+      queries.push(`DELETE ${sqlService.adminSchema}.${table} WHERE isLiveCheckForm = 0`)
```

Comment:
⚠️ If there are any checks with this checkForm the delete will fail - referential integrity constraints.

Reply:
@jon-shipley a validation rule is now included to prevent 'replacing' of the familiarisation check form when it is assigned.